### ISSUE TITLE
BUG: Wrong date when adding mentor note. 

### ISSUE THREAD

**Issue by gmai2015gm** [adamuremek/mentorship-program#74]
_**DESCRIPTION**_
When adding a note, the Date Created is off by a day.

_**STEPS TO REPRODUCE**_
1. When logged in as a mentor, click the "Add Note" button
2. Provide a title and a public note. 
3. Click "Create Note"
4. Observe the Date Created field of the new note. 

Notes:
- Here's a screenshot:
![image](https://github.com/adamuremek/mentorship-program/assets/60164202/e2ed0e50-c1b9-45f3-b4f1-8c432265eaed)
- This example was created around 9:30pm on 3/19/2024

**Comment by goupil-justin** [comment]
I'll take a look at it today.

**Comment by goupil-justin** [comment]
Had a few issues attempting to navigate to the page.

What branch are you working on?

**Comment by gmai2015gm** [comment]
It was on the dev server so Dev main. It might be something with the time zone on the server itself.

**Comment by goupil-justin** [comment]
That is most likely the issue and would affect everything time related.

I noticed that when a note was made during the demo this afternoon it displayed the correct time stamp. 

When I was looking last night I double checked the model. When the note is created the server grabs the current time stamp using date. So, I looked at how the note is retrieved. Everything I saw only reads the contents without modifying the note.

A solution would be adding a step where time is converted to the users local time zone. 

But the easiest solution would be to just make sure the server is configured to use Eastern Standard Time.
